src/event-manager.js

```diff
@@ -167,7 +167,20 @@ export default class EventManager {
       srcElement = handler;
       // If `event` is a map, call `on()` for each entry.
       for (const eventName in event) {
-        this._addEventHandler(eventName, event[eventName], srcElement);
+        this.on(eventName, event[eventName], srcElement);
+      }
+    }
+  }
+
+  // Register an event handler function to be called on `event`, then remove it
+  once(event, handler, srcElement) {
```

Comment:
Nit: you could DRY out the `on()` and `once()` implementations (keep them both part of the public API). Only difference is the `true` passed to `_addEventHandler`, correct?